src/api/test/functional/webui/spider_test.rb

```diff
@@ -33,6 +33,8 @@ def getlinks(baseuri, body)
       next unless link.port == baseuri.port
       link = link.to_s
       next if link =~ %r{/mini-profiler-resources}
+      # we do not really serve binary packages in the test environment
+      next if link =~ %r{/package/binary/download/*}
```

Comment:
https://www.regular-expressions.info/quickstart.html - just saying

Reply:
But it was like that on ~~stackoverflow~~ OBS codebase!

Reply:
el viejo grancanario ... :)